src/topology/vector_bundle/hom.lean

```diff
@@ -0,0 +1,267 @@
+topology on continuous linear maps between general topological vector spaces).  Likewise for tensor
+products of topological vector bundles, exterior algebras, and so on, where the topology can be
+defined using a norm on the fiber model if this helps.
+
+-/
+
+noncomputable theory
+
+open bundle set continuous_linear_map
+
+section defs
+variables {𝕜₁ 𝕜₂ : Type*} [normed_field 𝕜₁] [normed_field 𝕜₂]
+variables (σ : 𝕜₁ →+* 𝕜₂)
+variables {B : Type*}
+variables (F₁ : Type*) (E₁ : B → Type*) [Π x, add_comm_monoid (E₁ x)] [Π x, module 𝕜₁ (E₁ x)]
+variables [Π x : B, topological_space (E₁ x)]
+variables (F₂ : Type*) (E₂ : B → Type*) [Π x, add_comm_monoid (E₂ x)] [Π x, module 𝕜₂ (E₂ x)]
+variables [Π x : B, topological_space (E₂ x)]
+
+include F₁ F₂
+
+-- In this definition we require the scalar rings `𝕜₁` and `𝕜₂` to be normed fields, although
+-- something much weaker (maybe `comm_semiring`) would suffice mathematically -- this is because of
+-- a typeclass inference bug with pi-types:
+-- https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/vector.20bundles.20--.20typeclass.20inference.20issue
+/-- The bundle of continuous `σ`-semilinear maps between the topological vector bundles `E₁` and
+`E₂`. This is a type synonym for `λ x, E₁ x →SL[σ] E₂ x`.
+
+We intentionally add `F₁` and `F₂` as arguments to this type, so that instances on this type
+(that depend on `F₁` and `F₂`) actually refer to `F₁` and `F₂`. -/
+@[derive inhabited, nolint unused_arguments]
+def bundle.continuous_linear_map (x : B) : Type* :=
+E₁ x →SL[σ] E₂ x
+
+instance bundle.continuous_linear_map.add_monoid_hom_class (x : B) :
+  add_monoid_hom_class (bundle.continuous_linear_map σ F₁ E₁ F₂ E₂ x) (E₁ x) (E₂ x) :=
+by delta_instance bundle.continuous_linear_map
+
+variables [Π x, has_continuous_add (E₂ x)]
+
+instance (x : B) : add_comm_monoid (bundle.continuous_linear_map σ F₁ E₁ F₂ E₂ x) :=
+by delta_instance bundle.continuous_linear_map
+
+variables [∀ x, has_continuous_smul 𝕜₂ (E₂ x)]
+
+instance (x : B) : module 𝕜₂ (bundle.continuous_linear_map σ F₁ E₁ F₂ E₂ x) :=
+by delta_instance bundle.continuous_linear_map
+
+end defs
+
+variables {𝕜₁ : Type*} [nondiscrete_normed_field 𝕜₁] {𝕜₂ : Type*} [nondiscrete_normed_field 𝕜₂]
+  (σ : 𝕜₁ →+* 𝕜₂)
+
+variables {B : Type*} [topological_space B]
+
+variables (F₁ : Type*) [normed_group F₁] [normed_space 𝕜₁ F₁]
+  (E₁ : B → Type*) [Π x, add_comm_monoid (E₁ x)] [Π x, module 𝕜₁ (E₁ x)]
+  [topological_space (total_space E₁)]
+variables (F₂ : Type*) [normed_group F₂][normed_space 𝕜₂ F₂]
+  (E₂ : B → Type*) [Π x, add_comm_monoid (E₂ x)] [Π x, module 𝕜₂ (E₂ x)]
+  [topological_space (total_space E₂)]
+
+namespace topological_vector_bundle
+
+variables {F₁ E₁ F₂ E₂} (e₁ e₁' : trivialization 𝕜₁ F₁ E₁) (e₂ e₂' : trivialization 𝕜₂ F₂ E₂)
+variables [ring_hom_isometric σ]
+
+namespace pretrivialization
+
+/-- The coordinate change function between two trivializations of the vector bundle of
+continuous linear maps. -/
```

Comment:
It would help if you say in the docstring "Assume that `e₁ e₁'` are... and .... Then `continuous_linear_map_coord_change σ e₁ e₁' e₂ e₂' b` is ...". Indeed, the variables can not be seen in the statement (I mean, they are section variables), so being more explicit can only help.